### ISSUE TITLE
Adding Portable Storage for Flakes

### ISSUE THREAD

**Issue by rabbitscreams** [legobmw99/Allomancy#51]
I think it would be really useful to add a flake pouch or something similar to store large amounts of flakes on your person. Curio support would also be cool but not required!

**Comment by legobmw99** [comment]
This has been on my ideas list for a little while. I've never worked with inventories really, but I can't imagine it's too hard, so I'll try this next time I get some development time

**Comment by SonnyLonglegs** [comment]
How about making vials able to hold more than one type of metal? Like instead of 8 metals, you could fill it with 8 of 1 metal? It may not be the best solution, but it should be a simple one.

**Comment by legobmw99** [comment]
> How about making vials able to hold more than one type of metal? Like instead of 8 metals, you could fill it with 8 of 1 metal? It may not be the best solution, but it should be a simple one.

You can currently fill vials with all 16 metals at the same time

**Comment by SonnyLonglegs** [comment]
But can you fill them with 16x of one metal? That's what I meant, not sure how clear it was.

**Comment by legobmw99** [comment]
> 
> 
> But can you fill them with 16x of one metal? That's what I meant, not sure how clear it was.

No, and this won't be added. It would be confusing and not that useful

**Comment by Fictional-CJ** [comment]
What if there was a vial sash instead that stored the vials?
or is that a bad idea?

**Comment by legobmw99** [comment]
Wondering aloud: is this still that useful with bundles in the base game now?
